stream-chat-android-client/src/main/java/io/getstream/chat/android/client/setup/state/ClientState.kt

```diff
@@ -32,9 +33,9 @@ public interface ClientState {
     public val user: StateFlow<User?>
 
     /**
-     * If the client connection has been initialized.
+     * The state of the initialization process of the SDK.
      */
-    public val initialized: StateFlow<Boolean>
+    public val initialized: StateFlow<InitializationState>
```

Comment:
Can we implement it in a non-breaking way? Maybe we can add additional property `public val initializationState: StateFlow<InitializationState>` and deprecate the old one?